test/core/test_dlfcn_jspi_side.c

```diff
@@ -0,0 +1,7 @@
+#include <stdio.h>
+int test_wrapper(void);
+
+int f() {
+  printf("f\n");
```

Comment:
Could you call this function something with more meaning.  How about `side_module_function`.. or also use `sidey` is some other tests.

Then the printf could print that name too.

Reply:
Maybe `side_module_trampoline()` since it just calls back into the main module?